bugbot/rules/bugscore.py

```diff
@@ -0,0 +1,144 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbot import utils
+from bugbot.bzcleaner import BzCleaner
+
+
+class BugScore(BzCleaner):
+    def __init__(self):
+        super(BugScore, self).__init__()
+        self.ndups = self.get_config("number_dups")
+        self.votes = self.get_config("number_votes")
+        self.cc = self.get_config("number_cc")
+        self.see_also = self.get_config("number_see_also")
+        self.comments = self.get_config("number_comments")
```

Comment:
nit: It could be useful to add a comment to clarify that these thresholds are to highlight the values in the table, not to be used in the query.